x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/rules/50_query_rule_put.yml

```diff
@@ -0,0 +1,114 @@
+setup:
```

Comment:
do we need a `teardown` here where we remove the created rulesets? I noticed the other query rules yml tests are also missing a `teardown` section? is that taken care of somewhere else?

Reply:
I omitted these because of comments from [other PRs](https://github.com/elastic/elasticsearch/pull/96849#discussion_r1231305202) indicating that they're not needed and should be taken care of by the tests themselves.